src/javascript/app/ng-wig/ng-wig-plugin-adapter.component.spec.js

```diff
@@ -1,66 +0,0 @@
-describe('component: ngWigPlugin', () => {
```

Comment:
@robhicks I think it is preferable to have all tests along with the actual code. Please have a look at [Y197](https://github.com/bampakoa/angular-styleguide/tree/master/a1#style-y197) style in Angular 1 StyleGuide.